docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,790 @@
-* <<release-notes-8.0.0-rc2>>
-* <<release-notes-8.0.0-rc1>>
-* <<release-notes-8.0.0-beta1>>
-* <<release-notes-8.0.0-alpha2>>
-* <<release-notes-8.0.0-alpha1>>
+Review important information about the {kib} 8.x releases.
+
+* <<release-notes-8.2.0>>
+//* <<release-notes-8.1.3>>
+//* <<release-notes-8.1.2>>
+//* <<release-notes-8.1.1>>
+//* <<release-notes-8.1.0>>
+//* <<release-notes-8.0.1>>
+//* <<release-notes-8.0.0>>
+//* <<release-notes-8.0.0-rc2>>
+//* <<release-notes-8.0.0-rc1>>
+//* <<release-notes-8.0.0-beta1>>
+//* <<release-notes-8.0.0-alpha2>>
+//* <<release-notes-8.0.0-alpha1>>
 
 --
+
+[[release-notes-8.2.0]]
+== {kib} 8.2.0
+
+coming::[8.2.0]
+
+Review the following information about the {kib} 8.2.0 release.
+
+[float]
+[[breaking-changes-8.2.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+There are no breaking changes in {kib} 8.2.0.
+
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following: 
+
+{kibana-ref-all}/8.1/release-notes-8.1.0.html#breaking-changes-8.1.0[8.1.0] | {kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]
+      
+[float]
+[[features-8.2.0]]
+=== Features
+{kib} 8.2.0 adds the following new and notable features.
+
+Alerting::
+* Keep the number_of_scheduled_actions in event log {kibana-pull}128438[#128438]
+* Remove defaultRuleTaskTimeout and set ruleType specific timeout from kibana.yml {kibana-pull}128294[#128294]
+* Limit the executable actions per Rule execution {kibana-pull}128079[#128079] and {kibana-pull}126902[#126902]
+* Adds Cases to the Stack Management page {kibana-pull}125224[#125224]
+
+Dashboard::
+* Adds time slider control {kibana-pull}128305[#128305]
+* Adds Control group search settings {kibana-pull}128090[#128090]
+* Adds hierarchical chaining setting to Controls {kibana-pull}126649[#126649]
+* Adds options list API and validation system {kibana-pull}123889[#123889]
+
+Discover::
+Adds `Copy to clipboard` ability for column name of Document Explorer {kibana-pull}123892[#123892]
+
+Elastic Security::
+For the Elastic Security 8.2.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Lens & Visualizations::
+* Adds manual annotations in *Lens* {kibana-pull}126456[#126456]
```

Comment:
Should we call out it's a tech preview feature here?

Reply:
We call out the technical preview in the What's new and main Lens docs.